src/python/grpcio/tests/unit/_adapter/_intermediary_low_test.py

```diff
@@ -302,7 +302,8 @@ def testOneManyByteEcho(self):
     self._perform_echo_test([_BYTE_SEQUENCE])
 
   def testManyOneByteEchoes(self):
-    self._perform_echo_test(_BYTE_SEQUENCE)
+    self._perform_echo_test(
```

Comment:
I don't understand the change on this line. It's probably me not knowing something tricky or subtle. What?